server/area_manager.py

```diff
@@ -121,6 +121,15 @@ def remove_client(self, client: ClientManager.Client):
             if client.char_id != -1:
                 database.log_room('area.leave', client, self)
 
+        def client_can_additive(self, client: ClientManager.Client):
+            if self.last_ic_message is None:
```

Comment:
Seems like you forgot to commit something. `self.last_ic_message` should be defined in the constructor of `Area`.

Reply:
You angry banana. It's in there @oldmud0 ![image](https://user-images.githubusercontent.com/36182383/106018198-43dbcc00-608f-11eb-8748-ca86ef85e372.jpeg)

Reply:
...oh